docker/dropin_env_base/build_image.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Much better to use a system agnostic approach:
```suggestion
#!/usr/bin/env bash
```